tests/beaker_tests/netdev_stdlib/name_server_provider_defaults.rb

```diff
@@ -74,7 +74,7 @@
 
     # Expected exit_code is 0 since this is a vegas shell cmd.
     # Flag is set to true to check for absence of RegExp pattern in stdout.
-    cmd_str = UtilityLib.get_vshell_cmd('show running-config section name-server')
```

Comment:
@edwardpham I should have mentioned this yesterday but we can actually remove these steps all together since they are vsh related.  We are still using the `puppet resource` command to verify functionality.